### DISCUSSION TITLE
Log4J Vulnerability with Cmder (Console Emulator)

### DISCUSSION THREAD

**Discussion by bacmic** [cmderdev/cmder#2643] [Q&A] [answered]
Hello,

Our vulnerability mgmt team flagged a critical vulnerability in Log4J.

Is your technology affected by the Log4J vulnerability (also known as CVE-2021-44228 and CVE-2021-45046)?

**Comment by MartiUK** [comment, marked as answer]
No

**Reply by DRSDavidSoft** [reply]
This is correct.  Cmder does not use Log4J, nor any Java-based utilities that would depend on Log4J.
